shared/Dictionary.lua

```diff
@@ -0,0 +1,51 @@
+Dictionary = class(ValueStorage)
+
+function Dictionary:__init()
+    self:InitializeValueStorage(self)
+
+    self:SetValue("current_language", "en")
+end
+
+function Dictionary:AddLanguage(lang)
+	self:SetValue(lang, {})
+end
+
+function Dictionary:AddPhrase(lang, id, phrase)
+	local phrases = self:GetValue(lang)
+	if not phrases then
+		print("No language found with id " .. lang)
+	end
+
+	phrases[id] = phrase
+
+	self:SetValue(lang, phrases)
+end
+
+function Dictionary:SetLanguage(lang)
```

Comment:
Since we don't expect the current language to be changed during runtime, lets move this to the global config so it's easily configured.
Please make a class like this in napi/shared to store a config value for current language:
```
NapiConfig = class()

function NapiConfig:__init()
   self.Language = LanguageEnum.English
end

NapiConfig = NapiConfig()
```
Now we can access it with `NapiConfig.Language`